src/app/tests/TestCommandInteraction.cpp

```diff
@@ -58,8 +58,16 @@ namespace app {
 void DispatchSingleClusterCommand(chip::ClusterId aClusterId, chip::CommandId aCommandId, chip::EndpointId aEndPointId,
                                   chip::TLV::TLVReader & aReader, Command * apCommandObj)
 {
+    chip::app::CommandPathParams commandPathParams = { aEndPointId, // Endpoint
+                                                       0,           // GroupId
+                                                       aClusterId,  // ClusterId
+                                                       aCommandId,  // CommandId
+                                                       (chip::app::CommandPathFlags::kEndpointIdValid) };
     ChipLogDetail(Controller, "Received Cluster Command: Cluster=%" PRIx16 " Command=%" PRIx8 " Endpoint=%" PRIx8, aClusterId,
                   aCommandId, aEndPointId);
+
+    apCommandObj->AddStatusCode(&commandPathParams, Protocols::SecureChannel::GeneralStatusCode::kSuccess,
+                                Protocols::SecureChannel::Id, -Protocols::SecureChannel::kProtocolCodeSuccess);
```

Comment:
Why the `-` before the last arg?

Reply:
That is super weird, how compilation pass.... just fix this typo